### PR TITLE
feat(hacbs-1433): add dependabot

### PR DESCRIPTION
**Pull request by parasense**
Signed-off-by: Jon Disnard <jdisnard@redhat.com>

### CODE REVIEW

**Comment by johnbieren** [comment]
Is there a doc or something you followed to help you create this yaml? I found dependabot's website easily but there is a lot of documentation to go over and I am not sure what keyword to search for the actions added with this yaml

**Comment by parasense** [comment]
> Is there a doc or something you followed to help you create this yaml? I found dependabot's website easily but there is a lot of documentation to go over and I am not sure what keyword to search for the actions added with this yaml

I have been looking at this:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  # Maintain dependencies for GO modules.
+  - package-ecosystem: gomod
```
Comment (johnbieren) [inline .github/dependabot.yml:11]: Should the values here have quotes like they do in the stanza on line 5?

**Comment by scoheb** [comment]
I don't see automatic PR for this: https://github.com/redhat-appstudio/release-service/pull/40

I would expect to see this type of result when using dependabot: https://github.com/jenkinsci/jms-messaging-plugin/pull/241

**Comment by davidmogar** [comment]
Reopened by mistake. I don't agree on getting an automatic PR. You can generate a manual PR out of the alert. I think that's the best approach. Quoting:
```
Dependabot alerts
Receive alerts for vulnerabilities that affect your dependencies and manually generate Dependabot pull requests to resolve these vulnerabilities.
```

**Comment by scoheb** [comment]
you are right, the DoD from the M10 doc says:

> all of the team’s Github repos have Dependabot enabled, and admins/maintainers are getting alerts

But where are the alerts? maybe I missed some.

**Comment by davidmogar** [comment]
> you are right, the DoD from the M10 doc says:
> 
> > all of the team’s Github repos have Dependabot enabled, and admins/maintainers are getting alerts
> 
> But where are the alerts? maybe I missed some.

Alerts are configured per account. You can go to your profile and define how you want to get them. Also, you can click on the security alert. For some reason the one we got in the past is not there anymore, but I have the link to it in a conversation with @johnbieren, so it happened.

**Comment by davidmogar** [comment]
> Alerts are configured per account. You can go to your profile and define how you want to get them. Also, you can click on the security alert. For some reason the one we got in the past is not there anymore, but I have the link to it in a conversation with @johnbieren, so it happened.

It was: https://github.com/redhat-appstudio/release-service/security/dependabot/1

It seems I removed it:
```
The alert you requested (#1) has been withdrawn and is no longer available.  
```

**Pull request closed without merging**